examples/desktop/heatmap/heatmap.py

```diff
@@ -5,12 +5,15 @@
 """
 
 # test_example = true
+# sphinx_gallery_pygfx_docs = 'screenshot'
 
 import fastplotlib as fpl
 import numpy as np
+from wgpu.gui.offscreen import WgpuCanvas
 
+canvas = WgpuCanvas()
 
-fig = fpl.Figure()
+fig = fpl.Figure(canvas=canvas)
```

Comment:
remove

Reply:
this example has changed anyways in #511 so let's wait until that is merged, and then rebase this on that.

Reply:
I'll just remove the heatmap examples from being generated for now